packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Unreleased
+- [changed] Improved how Firestore handles idle queries to reduce the cost of
+  re-listening within 30 minutes.
```

Comment:
Do we want to go ahead and mention @var-const change as well?  Something like:

[changed] Improved offline performance with many outstanding writes.

Reply:
Done.